app/models/admin/users/police/user-list.js

```diff
@@ -108,9 +108,10 @@ function ViewModel (details, options, backNav, errors) {
 
   this.model = {
     backLink: backNav?.backLink || routes.index.get,
+    userCount: `${details.count} police officer${details.count > 1 ? 's' : ''}`,
     fieldset: {
       legend: {
-        text: 'Police officers with access to the Index',
+        text: 'Police officer access to the Index',
```

Comment:
I thought this was a bit picky TBH. Your version was fine :)

Reply:
pretty sure it changed as I generally copy and paste from the prototype, but nevermind 😄